### ISSUE TITLE
API Error

### ISSUE THREAD

**Issue by Arterter** [yandevelop/Bea#28]
When trying to upload one BeFake, I am getting API Error.

Bad Request, Bad Request, bad-request

**Comment by Arterter** [comment]
I think this is the error: {"error":"Bad Request","statusCode":400,"message":"Bad Request","errorKey":"bad-request","data":null,"fields":["caption"],"timestamp":"2023-11-12T02:42:45.233Z","requestId":"1307c16c014520e6f49251d5c3688868"}

can you help me here, thx

**Comment by yandevelop** [comment]
Try again, should still work.

**Comment by Arterter** [comment]
I have tried again and still gives me bad request? any clues ?

**Comment by yandevelop** [comment]
Bea probably hasn't fetched the access token it uses to post your BeReal so either BeReal itself have changed something in regards to access tokens or something's broken. Weird that it worked fine for me 8 hours ago. Have to look into it, which is going to happen tomorrow at earliest. Will update here if anything new

**Comment by yandevelop** [comment]
Looked into the requests now. I might have to include "bereal-signature" in the requests made by Bea. Will post a beta deb here asap

**Comment by yandevelop** [comment]
I haven't tested these for myself atm but if anyone can test please feel free to do so: https://easyupload.io/m/naju2t

**Comment by yandevelop** [comment]
Yup, i am aware of this. Try uploading without setting a comment. That's probably the whole issue
